packages/patternfly-3/react-console/src/DesktopViewer/vvFileGenerator.js

```diff
@@ -0,0 +1,22 @@
+import 'blob-polyfill';
```

Comment:
why is this required?

Reply:
Mostly due to IE: https://developer.mozilla.org/en-US/docs/Web/API/Blob#Browser_compatibility